src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -60,12 +78,13 @@
 
 	let filteredTokens: Token[] = [];
 	$: filteredTokens = isNullishOrEmpty(filterTokens)
-		? allIcrcTokens
-		: allIcrcTokens.filter(
-				({ name, symbol, alternativeName }) =>
-					name.toLowerCase().includes(filterTokens.toLowerCase()) ||
-					symbol.toLowerCase().includes(filterTokens.toLowerCase()) ||
-					(alternativeName ?? '').toLowerCase().includes(filterTokens.toLowerCase())
+		? allTokens
+		: allTokens.filter(
+				(token) =>
+					token.name.toLowerCase().includes(filterTokens.toLowerCase()) ||
```

Comment:
We compare token with their ids. Is there a specific reason why not to here?

Reply:
I actually kept the same searching logic: when the user search for a token, it checks the name, the symbol and, if there is, the alternative name

Reply:
I see, thanks for the feedback.